bin/ci-annotate-errors

```diff
@@ -12,4 +12,9 @@
 # ci-annotate-errors - Detect errors in junit xml as well as log files during
 # CI and find associated open GitHub issues in Materialize repository.
 
-exec "$(dirname "$0")"/pyactivate -m materialize.cli.ci_annotate_errors "$@"
+HOSTNAME=$(exec "$(dirname "$0")"/ci-builder run stable "$(dirname "$0")"/mzcompose --mz-quiet --find get-cloud-hostname run default --app-password-env-var "PRODUCTION_ANALYTICS_APP_PASSWORD")
+
+# remove the last line, which is the mzcompose result summary
+HOSTNAME=$(echo "$HOSTNAME" | head -n 1)
```

Comment:
This doesn't remove the last line, strictly speaking, but only keeps the first line.

Reply:
True, I will adjust the comment